### ISSUE TITLE
It Goes to 11... Quest Bugged

### ISSUE THREAD

**Issue by ToasterKetchup** [dalaranwow/dalaran-wow#5689]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: The harpoon does not spawn the defenders, thus making the quest not possible to complete.

**Expected behaviour**: The harpoon should spawn defenders.

**Steps to reproduce the problem**:

1. Obtain the quest.
2. Get on the harpoon with the quest item.

**Include proofs for this behaviour**
![untitled](https://user-images.githubusercontent.com/46654801/52809071-0eca4080-3055-11e9-80dc-43c6cf511578.png)

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**
The quest name is It Goes to 11...

You ride a Vrykul Harpoon Gun.
You must kill Dragonflayer Defenders, which do not spawn.

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by Lothloryen** [comment]
Just checked it and it worked fine, if anyone else experience an issue update this, please.
